diff --git a/SuiteSparse_config/CMakeLists.txt b/SuiteSparse_config/CMakeLists.txt
index 695a53897..017438898 100644
--- a/SuiteSparse_config/CMakeLists.txt
+++ b/SuiteSparse_config/CMakeLists.txt
@@ -123,12 +123,6 @@ else ( )
     message ( STATUS "No OpenMP and no clock_gettime available. Timing functions won't work." )
 endif ( )

-#-------------------------------------------------------------------------------
-# find the BLAS
-#-------------------------------------------------------------------------------
-
-include ( SuiteSparseBLAS )
-
 #-------------------------------------------------------------------------------
 # configure files
 #-------------------------------------------------------------------------------
@@ -241,16 +235,6 @@ else ( )
     endif ( )
 endif ( )

-# BLAS:
-if ( BLAS_FOUND )
-    # SuiteSparse_config does not itself require the BLAS.  It just needs to
-    # know which BLAS is going to be used by the rest of SuiteSparse so it
-    # can configure SuiteSparse_config.h properly.
-    message ( STATUS "BLAS libraries:      ${BLAS_LIBRARIES} ")
-    message ( STATUS "BLAS linker flags:   ${BLAS_LINKER_FLAGS} ")
-    message ( STATUS "BLAS include:        ${BLAS_INCLUDE_DIRS} ")
-endif ( )
-
 #-------------------------------------------------------------------------------
 # SuiteSparseConfig installation location
 #-------------------------------------------------------------------------------
